src/components/learner-credit-management/members-tab/tests/MembersTab.test.jsx

```diff
@@ -633,7 +640,7 @@ describe('<BudgetDetailPage />', () => {
     await waitForElementToBeRemoved(() => screen.queryByText('Removing (1)'));
     await waitFor(() => expect(screen.queryByText('There was an error with your request. Please try again.')).toBeInTheDocument());
   });
-  it('Remove toggle shows removed members', async () => {
```

Comment:
I removed this test because I didn't feel it was testing anything considering we're mocking the return statement from the useEnterpriseGroupMembersTableData hook

Reply:
it's been replaced by a download button test that ensures all filters are passed to the API call